tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+from pidgin.app import app as pidgin_app
+
+@pytest.fixture
```

Comment:
Something to keep in mind for pytest fixtures is the [scope](https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session). For mutable things you generally want the default, which is `function`, but for things which won't be changed throughout running the tests use `session` to save having to set up the same fixture multiple times for no reason. Also for fixtures that would be expensive to set up, you can think about how to keep these immutable (or resetting them between tests) to be able to use the `session` scope and save the cost of instantiating the same thing for hundreds of tests.

(Just to be clear—nothing wrong with this code specifically—just something to keep in mind as this codebase gets larger, or for working on other codebases)